tests/pw/utils/apiUtils.ts

```
@@ -356,6 +356,12 @@ export class ApiUtils {
         return responseBody;
     }
 
+    // update product
+    async updateProduct(productId: string, payload: object, auth?: auth): Promise<responseBody> {
+        const [, responseBody] = await this.put(endPoints.updateProduct(productId), { data: payload, headers: auth });
+        return responseBody;
+    }
```

Biome

[error] Duplicate class member name "updateProduct"



(lint/suspicious/noDuplicateClassMembers)